src/Microsoft.Identity.Client/IAuthenticationResult.cs

```diff
@@ -68,5 +68,10 @@ public interface IAuthenticationResult
         /// Gets the scope values returned from the service.
         /// </summary>
         IEnumerable<string> Scope { get; }
+
```

Comment:
Ok, let's follow up on Monday. I think both ios and android only return accessToken, expiresOn, User, and scope back.